src/lando/ui/jinja2/stack/partials/uplift-section.html

```diff
@@ -24,9 +24,11 @@ <h1>Uplifts</h1>
                         </span>
                     </button>
                 </div>
-                {% if uplift_assessment_link_form and uplift_assessment_link_form.has_assessments %}
+                {% if uplift_assessment_link_form %}
                     <div class="column is-narrow">
-                        <button class="button is-light link-assessment-open">
+                        <button class="button is-light link-assessment-open"
+                                {% if not uplift_assessment_link_form.has_assessments %}disabled{% endif %}
+                                title="{% if not uplift_assessment_link_form.has_assessments %}You have no previous uplift assessments to link{% endif %}">
```

Comment:
nit: This may be more readable done in one go:


```suggestion
                                {% if not uplift_assessment_link_form.has_assessments %-}
                                disabled
                                title="You have no previous uplift assessments to link">
                                {-% endif %}
```

(+/- linting (;)